app/views/plans/_plan_title.html.erb

```diff
@@ -1,8 +1 @@
-<h1><%= plan.title %></h1> 
-
-<% if plan.visibility == 'is_test' %>
-  <div class="roadmap-info-box">
-    <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
-    <span><%= _('This is a') %> <strong><%= _('test plan') %></strong>.</span>
-  </div>
-<% end %>
+<h1><%= plan.title %></h1> 
```

Comment:
Does this still need to be a partial? 
is the intention down the line to move back towards a conditionally different title?